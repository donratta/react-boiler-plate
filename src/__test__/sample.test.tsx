import * as React from 'react';
import { renderHook } from '@testing-library/react';
import { useEffect, useState } from 'react';
import * as renderer from 'react-test-renderer';


describe('Sample test jsx', () => {
    it('Should pass as a renderer', () => {
        const { result } = renderHook(() => {
            const [name, setName] = useState('')
            useEffect(() => {
                setName('Alice')
            }, [])

            return name
        });

        expect(result.current).toBe('Alice')
    });
    it('Should be the component rendered here', () => {
        const component = renderer.create(<div>Sample</div>);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
