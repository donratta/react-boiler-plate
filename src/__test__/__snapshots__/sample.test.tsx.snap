// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sample test jsx Should be the component rendered here 1`] = `
<div>
  Sample
</div>
`;
